### PR TITLE
[15.0] [MIG] product_category_active

### PR DESCRIPTION
**Pull request by tgrellety**

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]
Diff review / no test : lgtm.

thanks.

File: product_category_active/tests/test_product_category_active.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.exceptions import ValidationError
+from odoo.tests import SavepointCase
+
+
+class TestProductCategoryActive(SavepointCase):
```
Comment (ajaniszewska-dev) [inline product_category_active/tests/test_product_category_active.py]: as per https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0:

Replace SavepointCase by TransactionCase in tests, as they are now the same. The old one still exists as an alias, but a warning will arise, and next version will remove such alias.

**Comment by ajaniszewska-dev** [comment]
@tgrellety  small comment regarding test case, can you please fix pre commit as well? use  `pre-commit run --all `command for that and commit modified files.

**Review by i-vyshnevska — APPROVED** [review]

**Comment by Rad0van** [comment]
@tgrellety pre-commit is failing probably because of that SavepointCase not being replaced by TransactionCase. Would you please fix that so the PR could be merged? If you're not interested in continuing this someone else can take over.

File: product_category_active/tests/test_product_category_active.py (hunk 1 above)
Comment (Rad0van) [inline product_category_active/tests/test_product_category_active.py]: This needs to be fixed.

**Review by Rad0van — CHANGES_REQUESTED** [review]
Please fix the SavepointCase -> TransactionCase.

File: product_category_active/tests/test_product_category_active.py (hunk 1 above)
Comment (tgrellety) [reply product_category_active/tests/test_product_category_active.py]: Thank you for your feedback.
Can you take care of this ?

Comment (Rad0van) [reply product_category_active/tests/test_product_category_active.py]: Nope, I cannot change content of your PR. Sorry.

**Comment by Rad0van** [comment]
@tgrellety better, but still not OK. You need to do 2 things:
1) run `pre-commit run -a` to have the sources formatted correctly (that's why pre-commit here fails). Then push. Will create another commit.
2) Squash the commits those 2 commits into the previous one so only the one with message _[[MIG] product_category_active: Migration to 15.0]_ remains. Like `git rebase -i HEAD~3`

**Comment by Rad0van** [comment]
@tgrellety can you please do those last things to get this merged? Thank you.

File: product_category_active/models/product_category.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProductCategory(models.Model):
+    _inherit = "product.category"
+
+    active = fields.Boolean(
+        string="Active",
```
Comment (Rad0van) [inline product_category_active/models/product_category.py]: Unfortunately there is another one. You always need to check that the pre-coomit ends up without error. In this case it's complaining about these 2 lines. The second one is redundant as the same string would be rendered just from the field's name.

So please remove the line 12, rerun pre-commit locally to be sure everything else is fine. Then commit, squash with the previous one and should be fine to go.

**Comment by Rad0van** [comment]
Now it seems fine!

**Review by Rad0van — APPROVED** [review]
Looks fine, tested.

**Comment by tgrellety** [comment]
> Looks fine, tested.

Yes, thanks for your help @Rad0van !

**Review by victoralmau — APPROVED** [review]
Code and functionality OK.

**Pull request merged**